### ISSUE TITLE
Page missing a title (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#135]
1 occurrence ... one of the LSA pages does not have a title in the HTML head tag: https://newton.dlib.indiana.edu/newton/lsa/help/hs160.html

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [ ] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@wehooper : I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

If you plan to make local fixes, you will need to install the SiteImprove Plugin for your preferred browser, to test your changes.

**Comment by wehooper** [comment]
I have provided a <title> element for the <head> of 'hs160.html'. It was the only help file of thirty where the title was missing from its usual place.

We can close this when the revised LSA component can be downloaded from github/IUBLibTech.
